example/spec/setup/PopulateProvidersCollections.js

```diff
@@ -1,18 +1,86 @@
+'use strict';
 
 const { addProviders, addCollections } = require('@cumulus/integration-tests');
+const { s3 } = require('@cumulus/common/aws');
 const { loadConfig } = require('../helpers/testUtils');
+const fs = require('fs-extra');
 const config = loadConfig();
 
 const collectionsDirectory = './data/collections';
 const providersDirectory = './data/providers';
 
+/**
+ * Custom require to get the files from test data for upload to S3
+ *
+ * @param {Object} module - module
+ * @param {string} filename - filename
+ * @returns {undefined} - none
+ */
+function requireAsText(module, filename) {
+  module.exports = fs.readFileSync(filename, 'utf8'); // eslint-disable-line no-param-reassign
+}
+
+require.extensions['.PDR'] = requireAsText;
```

Comment:
With the big caveat that I haven't tested it yet, I think something approximately like this would get around having to do the module.exports stuff here.

```javascript
const s3data = [
  'pdrs/MOD09GQ_1granule_v3.PDR',
  'granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
  'granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf',
  'granules/MOD09GQ.A2016358.h13v04.006.2016360104606_ndvi.jpg'
];

/**
 * Upload a file from the test-data package to the S3 test data
 *
 * @param {string} file - filename of data to upload
 * @param {string} bucket - bucket to upload to
 * @returns {Promise<Object>} - promise returned from S3 PUT
 */
async function uploadTestDataToS3(file, bucket) {
  const testDataPath = path.dirname(require.resolve('@cumulus/test-data'));
  const filePath = path.join(testDataPath, file);
  const data = await fs.readFile(filePath, 'utf8');

  // This isn't quite right, I'm not sure what the key was supposed to be
  const key = file;

  return s3().putObject({
    Bucket: bucket,
    Key: `cumulus-test-data/pdrs/${key}`,
    Body: data
  }).promise();
}
```

I'd be happy to pair on this to see if we can use the `require.resolve` solution and not have to use the deprecated `module.exports`.